lux/lib/lux/integrations/discord/common.ex

```diff
@@ -0,0 +1,44 @@
+defmodule Lux.Integrations.Discord.Common do
```

Comment:
How about this?
```suggestion
defmodule Lux.Integrations.Discord do
```

Then we could just use `Discord.auth()` and similar things. I think it looks a bit cleaner.